#747 | decode trx input to show contract function call

Viterbo opened the pull request:
# Fixes #747 


[Deployment](https://deploy-preview-759--dev-mainnet-teloscan.netlify.app/)

## Description
This PR includes the solution for two of the three views that the new TransactionInputViewer component should offer. The Default view requires a refinement of the algorithms, so it was separated in this issue (#762) and will be scheduled apart.

The two views implemented are:
- Original: is the input as plain text
- Decoded: Is the Table with all parameters' types and values.

## Test Scenarios

- https://deploy-preview-759--dev-mainnet-teloscan.netlify.app/tx/0x3c81e5a0636da5d29f2eed4e0487dca3e6f36fe341821882c7d351f2de1bf48e
![image](https://github.com/telosnetwork/teloscan/assets/4420760/d38ef9c2-004f-4125-9f7b-e7e196403a55)
![image](https://github.com/telosnetwork/teloscan/assets/4420760/21fe8afe-c3ae-4d80-a36f-7f47c54a520e)

- https://deploy-preview-759--dev-mainnet-teloscan.netlify.app/tx/0x5abb5f2c39a12ccc0cd96c879d3ccd3d756ce7bfca369d72b655349d57de61a6
![image](https://github.com/telosnetwork/teloscan/assets/4420760/e4e6a01d-63bb-49a7-84d6-ce56ca1038c3)
![image](https://github.com/telosnetwork/teloscan/assets/4420760/c05ffde8-ef00-4821-bf8b-014636fbf9d4)

pmjanus commented:
<img width="1380" alt="image" src="https://github.com/telosnetwork/teloscan/assets/97489909/c562ba4a-934a-4376-837d-aeea0d996231">
We need the decoded view to allow for copy paste of values, it should all be visible or scrollable, I think you can try to make font smaller or have the input take more than one line.

Viterbo commented:
I changed the table to show multiline text on value cell.
Also added a copy to clipboard button on the top-right corner of the table.
![image](https://github.com/telosnetwork/teloscan/assets/4420760/ee6e5307-5aef-4c9f-ab10-e2b22b842f78)

Viterbo commented:
Copy to clipboard by line:

![image](https://github.com/telosnetwork/teloscan/assets/4420760/064bbc60-e330-410b-bfe1-ec312a5e1c10)


Fixed mobile design: 


![image](https://github.com/telosnetwork/teloscan/assets/4420760/471b4c94-174d-4149-902f-7a5b0f545a1d)
